migration/v0.32/README.md

```diff
@@ -35,6 +35,40 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 
 ## Prerequisites
 
+- If `.objectStorage.sync.enabled: true` and `.thanos.objectStorage.type: swift` or `.harbor.persistence.type: swift` then the rclone jobs will automatically use swift for source and destination buckets. You will need to create the application credentials for swift and add them into the `secrets.yaml`:
+
+    <details><summary>Create source application credentials</summary>
+
+    ```bash
+    source ${CK8S_CONFIG_PATH}/openrc.sh
+    source <(sops -d ${CK8S_CONFIG_PATH}/secret/<source-env-openstack-user>.sh)
+
+    openstack application credential create <env-name>-swift
+
+    sops ${CK8S_CONFIG_PATH}/secrets.yaml
+
+    objectStorage.swift.applicationCredentialID
+    objectStorage.swift.applicationCredentialSecret
+    ```
+
+    </details>
+
+    <details><summary>Create destination application credentials</summary>
+
+    ```bash
+    source ${CK8S_CONFIG_PATH}/openrc.sh
+    source <(sops -d ${CK8S_CONFIG_PATH}/secret/<destination-env-openstack-user>.sh)
```

Comment:
Would this not create the application credentials in the wrong region? Unless application credentials span regions I think you need to make sure you are in the correct region as well (sourcing openrc.sh should give you the source region).